pkg/semodule/seiface/interface.go

```diff
@@ -0,0 +1,49 @@
+package seiface
```

Comment:
nit: this module name seems quite redundant. Why not call this just `interface` instead? We know it's related to `semodule` since that's the parent package name anyway.